src/main/java/cinema/shop/model/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,6 +1,11 @@
 package cinema.shop.model.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
 public class CinemaHallRequestDto {
+    @NotNull
```

Comment:
checklist
`Can we use @NotNull annotation above the field with primitive data types? Be careful :)
`

Reply:
I think we [can](https://docs.oracle.com/javaee/7/api/javax/validation/constraints/NotNull.html)

Reply:
Ok, I got it. It will set field to default.